### DISCUSSION TITLE
New Preview of Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore Breaks TimerTrigger'd Functions

### DISCUSSION THREAD

**Discussion by ChiefInnovator** [Azure/azure-functions-dotnet-worker#1590] [Feedback]
All of our Timer triggered functions stop working when we reference Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore and configure it at startup.

**Comment by fabiocav** [comment]
Thanks, @ChiefInnovator 

This is being tracked by https://github.com/Azure/azure-functions-dotnet-worker/issues/1584 and will be fixed with the next drop.
